src/AppBundle/Bridge/Chameleon/Migration/Script/update-1535695117.inc.php

```diff
@@ -0,0 +1,57 @@
+<h1>Build #1535695117</h1>
+<h2>Date: 2018-08-31</h2>
+<div class="changelog">
+    - #50: add manufacturer name to table display
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_tbl_conf', 'de')
+    ->setFields([
+        //'name' => 'shop_article',
```

Comment:
I see that you try to improve legibility of the change, but I'd like to avoid code that is commented out. Suggestion: Use the name as condition in setWhereEquals() instead of the ID - the name is unique and serves equally well as a condition.